velox/exec/HashTable.cpp

```diff
@@ -937,7 +938,14 @@ void HashTable<ignoreNullKeys>::parallelJoinBuild() {
           return std::make_unique<bool>(true);
         }));
     VELOX_CHECK(!partitionSteps.empty());
-    buildExecutor_->add([step = partitionSteps.back()]() { step->prepare(); });
+    buildExecutor_->add([driverThreadCtx, step = partitionSteps.back()]() {
+      std::unique_ptr<ScopedDriverThreadContext> scopedDriverThreadContext =
```

Comment:
Can we let ScopedDriverThreadContext handle the nullptr case to keep this simple? Thanks!